test/cmdUtil.js

```diff
@@ -0,0 +1 @@
+require('../lib/cmdUtil.js').generateNewToken();
```

Comment:
You're right, this test is useless, it was only to run it in test mode and manually check it. We could remove it or enrich it.